### PR TITLE
Add workgroup level TileShape

### PR DESCRIPTION
**Pull request by aacostadiaz**
This PR adds the tile shape for the data computed by a workgroup.

### CODE REVIEW

File: include/cutlass/gemm/kernel/intel_pvc_gemm.hpp [hunk 1]
```diff
@@ -101,14 +101,10 @@ class GemmUniversal<
 
   static constexpr int SubgroupSize = CollectiveMainloop::SubgroupSize; // sub_group size
   static constexpr uint32_t MaxThreadsPerBlock = CollectiveMainloop::MaxThreadsPerBlock;
-  static constexpr uint32_t MinBlocksPerMultiprocessor = CollectiveMainloop::MinBlocksPerMultiprocessor;
-
-  static constexpr int num_sg = MaxThreadsPerBlock / SubgroupSize; // number of sub_groups per work group
 
   using DpasShape = typename CollectiveMainloop::DpasShape;
```
[the remark this answers is not in the document]
Comment (aacostadiaz) [reply include/cutlass/gemm/kernel/intel_pvc_gemm.hpp:106]: DpasShape is the 81616 shape. I renamed the rest to WorkgroupTileShape or SubgroupTileShape

File: include/cutlass/gemm/collective/intel_pvc_mma.hpp [hunk 2]
```diff
@@ -101,13 +101,14 @@ struct CollectiveMma<
...
+  using SubgroupTileShape = decltype(tile_shape(TiledMma()));
 
-  static constexpr uint32_t MaxThreadsPerBlock = get<0>(DpasShape()) * get<1>(DpasShape());
+  static constexpr uint32_t MaxThreadsPerBlock =
+          cute::size(WorkgroupTileShape{}) / cute::size(SubgroupTileShape{})* SubgroupSize;
 
-  static constexpr int FragsM = get<0>(TileDpasShape{}) / get<0>(DpasShape()); // A frags per sub_group
-  static constexpr int FragsN = get<1>(TileDpasShape{}) / get<1>(DpasShape()); // B frags per sub_group
-  static constexpr int FragsK = get<2>(TileDpasShape{}) / get<2>(DpasShape());
+  static constexpr int FragsM = get<0>(SubgroupTileShape{}) / get<0>(DpasShape()); // A frags per sub_group
```
Comment (mehdi-goli) [inline include/cutlass/gemm/collective/intel_pvc_mma.hpp:109]: Here also would it be possible to change the DpasShape -> AtomTileshape

Comment (aacostadiaz) [reply include/cutlass/gemm/collective/intel_pvc_mma.hpp:109]: That could be the copy atom or the mma atom. I can use `MmaAtomShape` or `DpasAtomShape`  instead

Inside the `TiledMma` struct it is defined as `using AtomShape_MNK  = typename MMA_Atom::Shape_MNK;`

Comment (mehdi-goli) [reply include/cutlass/gemm/collective/intel_pvc_mma.hpp:109]: Yes for MMA collective lets use mmaAtomShape and for copy part lets use copyAttomShape

**Review by mehdi-goli — APPROVED** [review]

File: include/cutlass/gemm/kernel/intel_pvc_gemm.hpp [hunk 3]
```diff
@@ -65,7 +65,8 @@ class GemmUniversal<
 
   // Mainloop derived types
   using CollectiveMainloop = CollectiveMainloop_;
-  using TileShape = typename CollectiveMainloop::TileShape;
+  using TileShape = typename CollectiveMainloop::WorkgroupTileShape;
+  using WorkgroupTileShape = TileShape;
```
Comment (mehdi-goli) [inline include/cutlass/gemm/kernel/intel_pvc_gemm.hpp:69]:
```suggestion
  using WorkgroupTileShape = typename CollectiveMainloop::WorkgroupTileShape;
```

**Review by mehdi-goli — APPROVED** [review]

**Pull request merged**